image-cache/src/image_cache.c

```diff
@@ -688,8 +689,12 @@ static void image_cache_run(void *args) {
             // Follow redirects to work with Facebook API et al
             curl_easy_setopt(request->handle, CURLOPT_FOLLOWLOCATION, 1L);
 
+            // Setting this one to avoid freeze issue with iOS when internet is slow
```

Comment:
can we add the Stackoverflow, thread as reference here in the comments ?

Reply:
sure